src/lib/elements/forms/inputCheckbox.svelte

```diff
@@ -27,19 +28,23 @@
 </script>
 
 <FormItem>
-    <Label {required} {optionalText} hide={!showLabel} for={id}>
-        {label}
-    </Label>
+    {#if label}
+        <Label {required} {optionalText} hide={!showLabel} for={id}>
+            {label}
+        </Label>
+    {/if}
 
     <div class="input-text-wrapper">
         <input
             {id}
             {disabled}
             {required}
+            {indeterminate}
             type="checkbox"
             bind:this={element}
             bind:checked={value}
-            on:invalid={handleInvalid} />
+            on:invalid={handleInvalid}
+            on:click />
```

Comment:
shouldn't `on:click` have a handler ?

Reply:
Nope, this forwards the event in case you need it